### DISCUSSION TITLE
Quick latex and Obsidian latex suite

### DISCUSSION THREAD

**Discussion by mxxxyia** [artisticat1/obsidian-latex-suite#339] [Q&A] [answered]
Are these two plugins compatible? are there any known conflicting commands? I've just started out w obsidian (first yr maths student), so is there anything i should know about enabling both plugins simultaneously?

**Comment by bdm-k** [comment, marked as answer]
I think enabling both plugins simultaneously will lead to a lot of unexpected behaviors. The [README](https://github.com/joeyuping/quick_latex_obsidian) of Quick Latex says that it is no longer maintained and recommends using Latex Suite, so use Latex Suite only.
